docs/sphinx/using/backends/sims/mqpusims.rst

```diff
@@ -1,5 +1,7 @@
-Multi-Processor Platforms
----------------------------------------------------
+
+Multi-QPU (mqpu)
```

Comment:
Maybe 'Multiple Simulated QPUs'? Trying to differentiate it from the next title (line 20)